packages/react-composites/src/composites/CallComposite/adapter/CallingSoundSubscriber.ts

```diff
@@ -53,8 +76,34 @@ export class CallingSoundSubscriber {
       callEndedSound = new Audio(sounds?.callEnded?.path);
       callEndedSound.preload = 'auto';
     }
+    let callRingingSound: HTMLAudioElement | undefined;
+    if (sounds?.callRinging) {
+      callRingingSound = new Audio(sounds?.callRinging?.path);
+      callRingingSound.preload = 'auto';
+    }
     return {
-      callEndedSound
+      callEndedSound,
+      callRingingSound
     };
   }
 }
+
+/**
+ * Helper function to allow the calling sound subscriber to determine when to play the ringing
+ * sound when making an outbound call.
+ */
+const playRingingSound = (call: CallCommon, locator: CallAdapterLocator): boolean => {
```

Comment:
nit: rn `playRingingSound` to something like `shouldPlayRingingSound` as this doesn't do the actual Play